source/Keyboard.cpp

```diff
@@ -46,6 +48,12 @@ Keyboard::Keyboard(struct wlr_input_device *device)
   shortcuts["Alt+Left"] = {"Switch focus left", [](){ Commands::switch_focus_left(); }};
   shortcuts["Alt+Down"] = {"Switch focus Down", [](){ Commands::switch_focus_down(); }};
   shortcuts["Alt+Right"] = {"Switch focus Right", [](){ Commands::switch_focus_right(); }};
+  shortcuts["Ctrl+Alt+Down"] = {"Switch workspace", [](){ Commands::switch_workspace(true); }}; // forward
```

Comment:
Maybe put "forward" in the workspace name? If it sounds wonky you could put it in parenthesis